test/e2e_node/jenkins/conformance/conformance-jenkins.sh

```diff
@@ -22,15 +22,15 @@ set -x
 
 : "${1:?Usage test/e2e_node/jenkins/conformance-node-jenkins.sh <path to properties>}"
 
-. $1
+. "${1}"
```

Comment:
why change:
```
In ./test/e2e_node/jenkins/conformance/conformance-jenkins.sh line 25:
. $1
  ^-- SC2086: Double quote to prevent globbing and word splitting.
```